Modules/Core/Common/include/itkAnatomicalOrientation.h

```diff
@@ -0,0 +1,584 @@
+                             CoordinateEnum::AnteriorToPosterior,
+                             CoordinateEnum::LeftToRight>,
+    SAL = m_OrientationValue<CoordinateEnum::SuperiorToInferior,
+                             CoordinateEnum::AnteriorToPosterior,
+                             CoordinateEnum::LeftToRight>,
+
+    PIR = m_OrientationValue<CoordinateEnum::PosteriorToAnterior,
+                             CoordinateEnum::InferiorToSuperior,
+                             CoordinateEnum::RightToLeft>,
+    PSR = m_OrientationValue<CoordinateEnum::PosteriorToAnterior,
+                             CoordinateEnum::SuperiorToInferior,
+                             CoordinateEnum::RightToLeft>,
+    AIR = m_OrientationValue<CoordinateEnum::AnteriorToPosterior,
+                             CoordinateEnum::InferiorToSuperior,
+                             CoordinateEnum::RightToLeft>,
+    ASR = m_OrientationValue<CoordinateEnum::AnteriorToPosterior,
+                             CoordinateEnum::SuperiorToInferior,
+                             CoordinateEnum::RightToLeft>,
+    PIL = m_OrientationValue<CoordinateEnum::PosteriorToAnterior,
+                             CoordinateEnum::InferiorToSuperior,
+                             CoordinateEnum::LeftToRight>,
+    PSL = m_OrientationValue<CoordinateEnum::PosteriorToAnterior,
+                             CoordinateEnum::SuperiorToInferior,
+                             CoordinateEnum::LeftToRight>,
+    AIL = m_OrientationValue<CoordinateEnum::AnteriorToPosterior,
+                             CoordinateEnum::InferiorToSuperior,
+                             CoordinateEnum::LeftToRight>,
+    ASL = m_OrientationValue<CoordinateEnum::AnteriorToPosterior,
+                             CoordinateEnum::SuperiorToInferior,
+                             CoordinateEnum::LeftToRight>
+
+  };
+
+
+  /** \brief Initialize with CoordinateEnum's from separate axes.
+   *
+   * If multiple CoordinateEnums are from the same axes then the Orientation value is INVALID.
+   */
+  constexpr AnatomicalOrientation(CoordinateEnum primary, CoordinateEnum secondary, CoordinateEnum tertiary)
+    : m_Value(
+        SameOrientationAxes(primary, secondary) || SameOrientationAxes(primary, tertiary) ||
+            SameOrientationAxes(secondary, tertiary)
+          ? PositiveEnum::INVALID
+          : static_cast<PositiveEnum>(
+              (static_cast<uint32_t>(primary) << static_cast<uint8_t>(CoordinateMajornessTermsEnum::PrimaryMinor)) +
+              (static_cast<uint32_t>(secondary) << static_cast<uint8_t>(CoordinateMajornessTermsEnum::SecondaryMinor)) +
+              (static_cast<uint32_t>(tertiary) << static_cast<uint8_t>(CoordinateMajornessTermsEnum::TertiaryMinor))))
+  {}
+
+
+  constexpr AnatomicalOrientation(PositiveEnum toOrientation)
+    : m_Value(toOrientation)
+  {}
+
+
+  constexpr AnatomicalOrientation(NegativeEnum fromOrientation)
```

Comment:
Please consider declaring those constructors `explicit` (to avoid implicit type conversion).

Reply:
OK, so you did consider 😃